keanu-project/src/main/java/io/improbable/keanu/tensor/dbl/DoubleTensor.java

```diff
@@ -41,6 +41,14 @@ static DoubleTensor create(double... values) {
         return create(values, 1, values.length);
     }
 
+    static DoubleTensor createColumnVector(double... values) {
+        return create(values, values.length, 1);
+    }
+
+    static DoubleTensor createRowVector(double... values) {
```

Comment:
If there is going to be a `createColumnVector`, I think there should be a `createRowVector`. This makes it more clear what is being created, rather than relying on the knowing that `create(double...)` creates a column vector.

Reply:
Resolved offline, will take this out for now. If it starts to be needed in more places then we can add this in.